### DISCUSSION TITLE
Github Action suddenly fails

### DISCUSSION THREAD

**Discussion by mkjsix** [community/community#25733] [Actions]
<p>Hi,</p>
<p>We do have a Github Action working since a long time that is now failing:</p>
<aside class="onebox githubactions" data-onebox-src="https://github.com/SoftInstigate/restheart/actions/runs/1707228597">
  <header class="source">

      <a href="https://github.com/SoftInstigate/restheart/actions/runs/1707228597" target="_blank" rel="noopener nofollow ugc">github.com</a>
  </header>

  <article class="onebox-body">
    <div class="github-row">
  <div class="github-icon-container" title="Commit">

      <svg width="24" height="24" class="github-icon github-icon-failure" viewbox="0 0 24 24" aria-hidden="true"><path d="M1 12C1 5.925 5.925 1 12 1s11 4.925 11 11-4.925 11-11 11S1 18.075 1 12zm8.036-4.024a.75.75 0 00-1.06 1.06L10.939 12l-2.963 2.963a.75.75 0 101.06 1.06L12 13.06l2.963 2.964a.75.75 0 001.061-1.06L13.061 12l2.963-2.964a.75.75 0 10-1.06-1.06L12 10.939 9.036 7.976z"></path></svg>

  </div>

  <div class="github-info-container">
    <h4>
      <a href="https://github.com/SoftInstigate/restheart/actions/runs/1707228597" target="_blank" rel="noopener nofollow ugc">:wrench: Update default buffer-size
</a>
    </h4>

      <div class="github-info">
        Build snapshot release <span class="github-run-number">#90</span>
      </div>
  </div>
</div>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p><code>Error: Failed to execute goal io.fabric8:docker-maven-plugin:0.33.0:start (start) on project restheart: I/O Error: [mongo:3.6]: Timeout after 60461 ms while waiting on url [http://0.0.0.0:27017](http://0.0.0.0:27017/) -&gt; [Help 1]</code></p>
<p>Basically we have a new unexpected behaviour with the docker-maven-plugin which has never happened before, we didn’t change this piece of POM since several months.</p>

**Comment by airtower-luna** [comment]
<p>Do you have any idea where the <code>0.0.0.0</code> in the URL comes from, or what should be there? I don’t know what the cause is, but that URL is definitely broken, so it’s probably a good place to start looking. :sweat_smile:</p>

**Comment by mkjsix** [comment]
<p>Thanks <a class="mention" href="/airtower-luna">@airtower-luna</a> but that doesn’t look to be the problem. It is just the maven plugin binding to all internet interfaces. As you can verify, this pipeline was working fine and untouched for at least two years now, and the build still works perfectly fine on our laptops.</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/SoftInstigate/restheart/runs/4827019192">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/SoftInstigate/restheart/runs/4827019192" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>

  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/365382/181092935-a1f32fc8-fe62-41ac-8bbd-1f0c1fe55a45.png" class="thumbnail onebox-avatar" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png">

<h3><a href="https://github.com/SoftInstigate/restheart/runs/4827019192" target="_blank" rel="noopener nofollow ugc">Build software better, together</a></h3>

  <p>GitHub is where people build software. More than 73 million people use GitHub to discover, fork, and contribute to over 200 million projects.</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<pre><code class="lang-auto">[INFO] Executed tasks
[INFO] 
[INFO] --- docker-maven-plugin:0.33.0:start (start) @ restheart ---
[INFO] DOCKER&gt; Pulling from library/mongo
[INFO] DOCKER&gt; Digest: sha256:146c1fd999a660e697aac40bc6da842b005c7868232eb0b7d8996c8f3545b05d
[INFO] DOCKER&gt; Status: Downloaded newer image for mongo:3.6
[INFO] DOCKER&gt; Pulled mongo:3.6 in 10 seconds 
[INFO] DOCKER&gt; [mongo:3.6]: Start container e382a63376dc
[INFO] DOCKER&gt; [mongo:3.6]: Waiting on url http://0.0.0.0:27017 with method GET for status 200.
[INFO] DOCKER&gt; [mongo:3.6]: Waited on url http://0.0.0.0:27017 787 ms
[INFO] 
[INFO] --- maven-antrun-plugin:1.8:run (start-server) @ restheart ---
[INFO] Executing tasks
</code></pre>

**Comment by mkjsix** [comment, marked as answer]
<p>Apparently, the pipeline has repaired itself…</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/SoftInstigate/restheart/runs/4845581723?check_suite_focus=true">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/SoftInstigate/restheart/runs/4845581723?check_suite_focus=true" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/304;"><img src="https://user-images.githubusercontent.com/365382/181092936-affc2c9e-8097-47ff-a3b0-1acd6297ca61.png" class="thumbnail" width="690" height="304" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/c/9/c919e1c91aeaa52ce2ebd28f012e808d17325986_2_690x304.png, https://aws1.discourse-cdn.com/github/optimized/3X/c/9/c919e1c91aeaa52ce2ebd28f012e808d17325986_2_1035x456.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/c/9/c919e1c91aeaa52ce2ebd28f012e808d17325986_2_1380x608.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/c/9/c919e1c91aeaa52ce2ebd28f012e808d17325986_2_10x10.png"></div>

<h3><a href="https://github.com/SoftInstigate/restheart/runs/4845581723?check_suite_focus=true" target="_blank" rel="noopener nofollow ugc">:wrench: Update default buffer-size · SoftInstigate/restheart@b025f43</a></h3>

  <p>RESTHeart - Instant REST, GraphQL and WebSocket API for MongoDB. - :wrench: Update default buffer-size · SoftInstigate/restheart@b025f43</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>I guess it was a transient error on Github Actions.</p>
